diff --git a/build/core/product.mk b/build/core/product.mk
index b13a38f..e99d0d4 100644
--- a/build/core/product.mk
+++ b/build/core/product.mk
@@ -374,8 +374,6 @@ $(strip \
         $(eval $(warning previous value: $($(strip $(1))_$(call rot13,$(v))))) \
         $(eval changed_variables := $(changed_variables) $(v))) \
    ) \
-  $(if $(changed_variables),\
-    $(eval $(error The following variables have been changed: $(changed_variables))),)
 )
 endef
 
