diff --git a/device/qcom-common/power/power.c b/device/qcom-common/power/power.c
index 238eb58..7010211 100644
--- a/device/qcom-common/power/power.c
+++ b/device/qcom-common/power/power.c
@@ -202,7 +202,6 @@ static void power_hint(__attribute__((unused)) struct power_module *module, powe
         case POWER_HINT_INTERACTION:
         case POWER_HINT_CPU_BOOST:
         case POWER_HINT_LAUNCH_BOOST:
-        case POWER_HINT_AUDIO:
         case POWER_HINT_SET_PROFILE:
         case POWER_HINT_LOW_POWER:
         break;
